### PR TITLE
changed minor errors

### PR DESCRIPTION
**Pull request by giwon2004**

### CODE REVIEW

**Comment by Misaki290** [comment]
IMG_20210515_113251

Shouldn't "Staus" be translated?
You left it blank.
If you don't/can't translate it, you should keep the word in English.
If you keep it blank, I don't think that it will appear in game.

**Comment by BruteDeity** [comment]
Misaki is right. if you leave that blank it wont show in the game. please fix this and any similar issue and let me know when you're ready for me to take a look at it

**Comment by DexterHuang** [comment]
why do we want to remove the word status in korean version ?@@

**Comment by Misaki290** [comment]
> why do we want to remove the word status in korean version ?@@

Ask that guy. Not me. I am not korean nor him to know what he is thinking.

File: contribution/lang/ko.json [hunk 1]
```diff
@@ -1034,7 +1034,7 @@
  	 	 	 	"eng": "Street Cred"
  	 	 	},
  	 	 	"status": {
- 	 	 	 	"trans": "스탯",
+ 	 	 	 	"trans": "",
```
Comment (aciddust) [inline contribution/lang/ko.json:1037]: I think it should be '상태', not "스탯" on this part.
image

right?

@giwon2004   how do you think 'bout this?

**Comment by DexterHuang** [comment]
let me close this first @@ seems like it shouldn't be removed

**Pull request closed without merging**